util/src/main/java/io/grpc/util/OutlierDetectionLoadBalancer.java

```diff
@@ -211,9 +213,20 @@ protected Helper delegate() {
     public Subchannel createSubchannel(CreateSubchannelArgs args) {
       // Subchannels are wrapped so that we can monitor call results and to trigger failures when
       // we decide to eject the subchannel.
-      OutlierDetectionSubchannel subchannel = new OutlierDetectionSubchannel(
-          delegate.createSubchannel(args));
-
+      HealthUtil.HealthCheckingListener hcListener = args.getOption(HEALTH_LISTENER_ARG_KEY);
+      HealthUtil.ChainedHealthListener outlierListener = null;
+      if (hcListener != null ) {
+        HealthUtil.ChainedHealthListener chainedHealthListener =
+            new HealthUtil.ChainedHealthListener(hcListener);
+        args = args.toBuilder()
+            .addOption(HEALTH_LISTENER_ARG_KEY, chainedHealthListener).build();
+        if (hcListener.getGeneration() > 0) {
+          outlierListener = chainedHealthListener;
+        }
+      }
+      Subchannel originalSubchannel = delegate.createSubchannel(args);
+      OutlierDetectionSubchannel subchannel = new OutlierDetectionSubchannel(originalSubchannel,
```

Comment:
Is the wrapped subchannel stored anywhere?

Reply:
Nope. It is just tracked by outlier detection as it was. Nothing changed regarding storing is anywhere.